client/src/modules/payment/models.ts

```diff
@@ -1,5 +1,10 @@
 import { ErrorI } from '@/modules/error/store/actions'
 
+export const PaymentStatus = {
```

Comment:
If this type is the same as SbcPaymentStatus, why have both?

Reply:
The codes are different on both sides for various reasons including that a payment record on their end is not 100% analagous to a payment on ours. Our payment only references thee SbcPayment. If we need the SbcPayment we do not get it from our side, we always make the API call as we cannot control what happens on the SbcPayment side of things.

Reply:
I agree with that approach. The 2 types just look very similar to me :)